app/javascript/packages/document-capture/components/tip-list.tsx

```diff
@@ -0,0 +1,22 @@
+import { useI18n } from '@18f/identity-react-i18n';
+
+interface TipListProps {
+  title: string;
+  items: string[];
+  translationNeeded?: boolean;
```

Comment:
It doesn't look like this is `true` anywhere? I'd lean towards removing this boolean and the logic that depends on it:
- It seems unlikely that we'd need to use this component from anywhere that doesn't already have the `t(...)` function available.
- It adds complexity to have this option, and we don't need it yet.

Reply:
@charleyf , cleaned up, originally i was trying to dynamically generate the i18n keys, but it seems translation does not work with dynamically loaded keys somehow, 🤔 .

Reply:
Interesting, I suspect this has to do with the nature of hooks, but would be happy to take a look if you run into it again.